### ISSUE TITLE
queryTimeout is not explicitly passed to goroutine

### ISSUE THREAD

**Issue by atc0005** [atc0005/dnsc#96]
While reviewing the code as a template for another project I'm working on, I noticed that the `queryTimeout` value is referenced from within the goroutine from the outer scope:

https://github.com/atc0005/dnsc/blob/5e60e9a4e4ec0d26570d4209b39f3ea9aa284a47/cmd/dnsc/main.go#L48

https://github.com/atc0005/dnsc/blob/5e60e9a4e4ec0d26570d4209b39f3ea9aa284a47/cmd/dnsc/main.go#L86

Since the value is the same for each routine (and isn't changing) this is *probably* fine, but if I refactor the anonymous function to place it elsewhere it won't have knowledge of that variable.

In short, that variable should probably be passed in as an argument.
